docs/application/vscode-ext/web.md

```diff
@@ -406,6 +377,93 @@ VS Code Extension for Tizen Web supports intelliSense. You can use these feature
 
   ![Document Viewer](media/tizen_web23.gif)
 
+## Tizen Web with Design Editor
+
+Visual Studio Code Extension for Tizen Web that supports WYSIWYG Design Editor features such as **Preview**, **JavaScript Assistant**, **Structure View**, **New Page**, **HTML Assistant** , **Undo** and **Redo**.
```

Comment:
Please remove "that" as given in the previous comment, https://github.com/Samsung/tizen-docs/pull/1016#discussion_r375615982.

Reply:
done